src/include/distributed/extended_op_node_utils.h

```diff
@@ -28,6 +28,7 @@ typedef struct ExtendedOpNodeProperties
 	bool hasNonPartitionColumnDistinctAgg;
 	bool pullDistinctColumns;
 	bool pushDownWindowFunctions;
+	bool pullUpIntermediateRows;
```

Comment:
This seems like the wrong place to store this since it isn't being set by `BuildExtendedOpNodeProperties`. May make sense to push this logic further back in planner, especially since it seems like this can handle window function pull up, but the decision is currently tied to aggregation push down support, which is all in multi_logical_optimization

Reply:
For window functions we would still want to push down GROUP BY.